### DISCUSSION TITLE
what's the roadmap?

### DISCUSSION THREAD

**Discussion by YAGregor** [vitalik/django-ninja#649] [General]
I like this project over rest-framework in some part, especially it is typed. I noticed that there is still many different between ninja and rest-framework not just in code  but also in design and practice, eg:
* rest-framework has various configable backends
* restful-framework's serializers are designed around django model, it can create/update model and serialize data to model, 
* rest-framework has viewsets, filters...

will ninja be more like rest-framework for fast CRUD or fastapi for small project？

I'd like to see ninja be more like rest-framework, I suggest just copy design from rest-framework and fix mistakes in rest-framework like too inherit.

**Comment by sebastian-philipp** [comment]
Regarding your point with the serializers, I honestly enjoy working with ninja much more that with DRF when it comes to building the API. In my experience it is much easier to understand. Especially for big projects.

Regarding your point with classed based views. We have https://github.com/vitalik/django-ninja/issues/15 and I'd recommend reading https://github.com/vitalik/django-ninja/issues/15#issuecomment-1030673844
